frontend/src/metabase/query_builder/components/AlertModals/AlertModals.jsx

```diff
@@ -566,7 +567,29 @@ export const AlertSettingToggle = ({
   </div>
 );
 
-export function AlertEditSchedule({ alertType, schedule, onScheduleChange }) {
+function getSchedulePickerSendTimeText(alert) {
+  const channels = alert.channels.filter(channel => channel.enabled);
+  const [channel] = channels;
+  if (channels.length === 0) {
+    return;
+  }
+  if (channels.length === 2) {
+    return `Emails and Slack messages will be sent at`;
```

Comment:
I believe we should use t`` to translate these strings and kick off the translation process

Reply:
Nice catch, missed this